scienceplots/styles/color/discrete-rainbow/discrete-rainbow-1.mplstyle

```diff
@@ -0,0 +1,5 @@
+# Discrete rainbow color scheme with 1 colors
+# from Paul Tot's website: https://personal.sron.nl/~pault/
```

Comment:
```suggestion
# from Paul Tol's website: https://personal.sron.nl/~pault/
```
I see we always had this typo in the code... If you want to address that here it's fine, if not, I can do that later.

Reply:
I can also do that in my code, but I would suggest not doing that on everything else as then the PR would become overloaded with changes to almost all files

Reply:
Yeah, not a blocker.